mitodl/edx-sga
# Fixed template render on py3 - #CRI-175

#### What are the relevant tickets?
[Trello Ticket](https://trello.com/c/Z32kIoOx/13-sga-bad-character-encoding-when-run-on-python3-on-master)

#### What's this PR do?
Adds a `decode` step to template loading method that was missed earlier and resulted in bytes being printed directly rather than a unicode string.

#### How should this be manually tested?
Enable SGA in your deployment and add a unit in a course. Visit the unit both in the studio and LMS and verify no more `b'\n\n'` is printed across the template.

#### Screenshots (if appropriate)
Before
<img width="1580" alt="image" src="https://user-images.githubusercontent.com/6387678/79981055-797ad580-84bd-11ea-816a-80169c026aff.png">

After
![Screenshot 2020-04-22 at 5 22 19 PM](https://user-images.githubusercontent.com/6387678/79981264-d5455e80-84bd-11ea-963e-2eeef9af544c.png)
![Screenshot 2020-04-22 at 5 25 50 PM](https://user-images.githubusercontent.com/6387678/79981592-5e5c9580-84be-11ea-8b55-11ffbb360472.png)
